tests/conftest.py

```diff
@@ -35,7 +35,9 @@ def app():
 
 @pytest.fixture
 def testapp(app):
-    return TestApp(app)
+    _app = TestApp(app)
+    _app.authorization = ('Basic', ('username', 'password'))
```

Comment:
My concern is that we'll forget this is here and then tests will pass without explicitly adding the auth. I'd rather a test fail until I put in Auth I think. But it's not a huge deal.